strongbox-data-service/src/main/java/com/orientechnologies/orient/object/jpa/OJPAObjectDatabaseTxEntityManagerFactory.java

```diff
@@ -68,14 +94,14 @@ private EntityManager createEntityManager(final OJPAProperties properties)
     @Override
     public void close()
     {
-        databasePool.close();
+        dataSource.close();
```

Comment:
@fuss86 are you sure that we heed to close all `dataSoruce` here? I mean that it will close underling `OrientDB` judging by the source code.

Reply:
Yes, I believe so it's absolutely OK here. 

https://docs.oracle.com/html/E13946_03/ejb3_overview_emfactory_close.html

`EntityManagerFactory.close` will be invoked when the application shutdown has been invoked.

Reply:
oh, seems that I mixed it up with the `EntityManager`
you are certainly right of course, thanks :)